agent/src/beerocks/slave/agent_db.h

```diff
@@ -0,0 +1,78 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include <iostream>
+#include <memory>
+#include <mutex>
+
+namespace beerocks {
+
+/**
+ * @brief AgentDB is a class used by all main Agent threads on the Agent process to store the Agent
+ * common data.
+ * The class is implemented using singleton design pattern.
```

Comment:
```suggestion
 * @brief AgentDB is a class used by all main Agent threads on the Agent process to store the Agent
 * common data.
 *
 * The class is implemented using singleton design pattern.
```
(otherwise the entire paragraph will be part of the `@brief`.